### DISCUSSION TITLE
considering-replacing-disqus-with-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#1] [Announcements]
# considering-replacing-disqus-with-giscus/

In this post I'm soliciting opinions-should I pay to keep using Disqus for comments, or should I switch to an open-source alternative that uses GitHub

https://andrewlock.net/considering-replacing-disqus-with-giscus/

**Comment by emyfreya** [comment]
I guess you made the switch. I had to disable Seven JSON Viewer on Microsoft Edge to comment, weird.

**Reply by andrewlock** [reply]
Thanks, yeah, I switched across last night (blog post incoming 🙂). That's very strange about the extension causing issues, a bit concerning for the extension to be honest, if it's only supposed to be a viewer 😄

**Reply by emyfreya** [reply]
I love the change though ! :smile:
It is concerning !

Here's how the extension is configured in the first place :
![img](https://i.postimg.cc/jqWFJVNg/msedge-WIOz-TYLYui.png)
And the results :
![img2](https://i.postimg.cc/pyb8m58H/msedge-42jd-Sq-P8-Ts.png)

**Comment by DavidDeSloovere** [comment]
Giscus looks great. Your audience will have a github account, so logical decision in your case. Thanks for all your great posts.

**Comment by ricardodemauro** [comment]
Awesome!

**Comment by garywoodfine** [comment]
This looks a great option!  I was completely unaware that this actually existed!
For me though I might have to try knock together a WordPress plugin to enable this,  (yes, I'm one of those that uses WordPress for my blogging platform :-) )

In theory I am supposed to make money from my Disqus, but really its completely neglible and I don't think I've ever had a pay out!  I have been tying to keep tracking down completely on my website and I don't even use Google Analytics so this looks like a great option, also considering that 80% of my audience will generally have a Github account and I already send a vast majority of my traffic to Github anyway in terms of code sample repo's etc.  Option does make a lot of sense!

Thanks for bringing it to my attention

**Comment by eric-stanley** [comment]
Apt choice I guess. Another advantage of using giscus over utterance is it provides custom styles which can be used to match your site! And last but not least, giscus is actively maintained than utterance and has sponsorship from vercel.

I switched to giscus too!

**Comment by thedavecarroll** [comment]
Thanks for this article. It was immensely helpful in my decision for my own blog.

In fact, I'm writing an article on how I refreshed the tech stack of my site and I'll be linking to your Twitter profile and this blog page (assuming you are okay with it). Do you have any specific analytics links that you want me to include, like campaigns or marketing tags?

Thanks again.

**Reply by andrewlock** [reply]
Great to hear it, I look forward to reading it 🙂

[a comment by an automated account is not shown]

**Comment by btbarutcu** [comment]
I use Omni Comments as an alternative to Disqus on my website.
